themes/default/content/blog/organization-access-tokens/index.md

```diff
@@ -0,0 +1,60 @@
+
+  1. Enterprise customers using SSO/SAML/SCIM, as it enables them to create Pulumi Service tokens not tied to identities within their identity provider (IdP)
+  2. Garnering programmatic access for continuous integration and continuous delivery (CI/CD) tools and other automated processes
+
+We're excited to deliver one of the most [requested features from the community](https://github.com/pulumi/service-requests/issues/16) and a top ask from some of the largest customers using the Pulumi Service. The need to use Personal Access Tokens to manage automation and organization-specific resources can lead to some friction. When a user is deprovisioned or otherwise leaves the organization, their tokens in use no longer grant access and pipelines can become broken. Additionally, since Personal Access Tokens grant access to all resources and organizations they have access to, using personal tokens in organization automation pipelines leads to the pipeline having more privileged access than is necessary. We want to enable our customers to use the principle of least privilege. At launch, Organization Access Tokens will have write permissions to stacks in an Organization. We plan to continue to expand options for scoping this access based on feedback from our customers.
+
+See below what the experience will look like in the Pulumi Service Console:
+
+![Gif of tokens navigation](nav-org-tokens.gif)
+
+Users can now create an Organization Access Token in the Service Console, give it a unique human-readable name (i.e. jenkins-ci-dev) and then use it to gain access to their Pulumi Service Organization. Only Organization Admins can create and manage Organization Access Tokens. As a security consideration for our customers, Organization Access Tokens cannot create other Organization Access Tokens. When a token is created they are provided a one-time token value. These tokens can only be deleted as an explicit action, either by clicking a button in the console or calling an API to delete the token. Creating or deleting an Organization Access Token will be logged as an event in the [Audit Logs](https://www.pulumi.com/docs/intro/pulumi-service/audit-logs).
+
+To determine which actions have been taken by which Organization Access Tokens, and to give greater clarity into specifically which tokens are conducting what operations, we have expanded Audit Log events to include information about the Organization Access Token that took the action. As an organization can have multiple Organization Access Tokens, this is a necessary step to ensure that accurate auditing can be done in the event that a token is compromised or is being used maliciously.
+
+See below how Organization Access Tokens will appear in Audit Logs:
+![Gif of tokens in Audit Logs](audit-logs-org-tokens.gif)
```

Comment:
Adding an empty line here will apply the right margin between paragraph text and image (right now, it's just a line break): 

```suggestion

![Gif of tokens in Audit Logs](audit-logs-org-tokens.gif)
```